stable/openvpn/values.yaml

```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: alpine
-  tag: 3.4
+  repository: jfelten/openvpn-docker
+  tag: latest
```

Comment:
Can you version your image, and is there a source repo you can add to Chart.yaml?